src/main/java/org/gatorgradle/util/Read.java

```diff
@@ -0,0 +1,27 @@
+package org.gatorgradle.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Read {
+
+
+  /**
+   * Read in reflection file
+   *
+   * @param fileName the path to the reflection file
+   * @return the reflection file in string with json escaped
+   */
+  public static String readFile(String fileName) throws IOException {
+    try {
+      String reflection = String.join("\n", Files.readAllLines(Paths.get(fileName)));
+      return StringUtil.jsonEscape(reflection);
```

Comment:
We may want to review this `jsonEscape` method because I'm not sure it was designed with markdown files in mind, and so we might need to do some research to come up with better regular expressions to use. Keep in mind through all of this that we cannot use any external library very easily, as it introduces complex plugin dependencies in Gradle.

Reply:
It seems like JSON escape for markdown is pretty much same with other text files, although I'm not entirely sure about that, so for now I'm still using the `jsonEscape` method you implemented